README.md

```diff
@@ -1,86 +1,11 @@
-## Instructors ##
-[Grant Erickson](https://github.com/GrantErickson) and [Meg Woodford](https://github.com/mmwoodfo)
-For general questions, we suggest asking in Teams so that others can help as well. We encourage people to work together and collaborate. Please feel free to help your fellow students out. Though it is possible to privately message people directly, as much as possible, we encourage open discussion on the main chat so that others can benefit as well.
-You can also send an e-mail to: [EWU-Instructors@IntelliTect.com](EWU-Instructors@IntelliTect.com) for final grade or general questions.
+**Assignment 1**
 
-## GitHub Information
-We will be using GitHub for all class lectures and assignments. All work will be submitted via GitHub pull requests.
-* Please sign up for a GitHub if you do not already have one. 
-* You can get added benefits by signing up for a free [GitHub Student Developer Pack](https://education.github.com/pack).  
-* **IMPORTANT** : Please fill out this **[form](https://docs.google.com/forms/d/e/1FAIpQLSdlpBMhof2gxL64vrqLg9o0Uc4di5lYejpo10XUBiMoyb-Xxg/viewform?usp=sf_link)** with your GitHub information. This is used by us so we can appropriately grade assignments.
+Azure URL:  https://brave-rock-03632d41e.5.azurestaticapps.net/
```

Comment:
almost missed the URL with all the changes to Readme lol

Reply:
We changed the readme a lot at the end lol.